docs/app/component-testing/angular/overview.mdx

```diff
@@ -20,7 +20,7 @@ sidebar_label: Overview
 
 ## Framework Support
 
-Cypress Component Testing supports Angular 17.2.0+.
+Cypress Component Testing supports Angular 18.0.0+.
```

Comment:
```suggestion
Cypress Component Testing supports Angular ^18.0.0 and ^19.0.0
```

using the same notation as used on https://angular.dev/reference/releases#actively-supported-versions and excluding Angular v20.0 to be consistent with the get-started table which says Angular 18-19

Reply:
Very true, especially since we aren't even sure the current mount package works OOTB with Angular 20

Reply:
amended into d26bc875d